Source/WebCore/page/ResizeObservation.cpp

```diff
@@ -105,17 +107,19 @@ FloatSize ResizeObservation::snappedContentBoxSize() const
 std::optional<ResizeObservation::BoxSizes> ResizeObservation::elementSizeChanged() const
 {
     auto currentSizes = computeObservedSizes();
+    if (!currentSizes)
+        return currentSizes;
```

Comment:
I think it’s better to return `{ }` or `std::nullopt` rather than `currentSizes` here.